mbs_results/staging/dfs_from_spp.py

```diff
@@ -0,0 +1,50 @@
+import json
+
+import boto3
+import pandas as pd
+import raz_client
+from rdsa_utils.cdp.helpers.s3_utils import load_json
+
+
+def dfs_from_spp(
```

Comment:
function name should have an action, `read_dfs_from_spp` perhaps? or something like this